bucket/age.json

```
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://age-encryption.org/",
+    "version": "1.0.0-beta2",
+    "description": "A simple, modern and secure encryption tool with small explicit keys, no config options, and UNIX-style composability.",
+    "license": {
+        "identifier": "BSD 3-Clause",
+        "url": "https://github.com/FiloSottile/age/blob/master/LICENSE"
+    },
```

Since this is a standard SPDX license, we don't need to point to their `LICENSE` text.

```suggestion
    "license": "BSD-3-Clause",
```
(Note that you'll then need to fix the manifest because GitHub suggestions cannot overwrite more than one line)

---

Updated with 6dd2ac5447ed4e6df2d309851883540f70b78411